sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -60,12 +60,9 @@ def tearDownClass(cls):
     standard -= set([coders.Coder,
                      coders.FastCoder,
                      coders.Base64PickleCoder,
-                     coders.FloatCoder,
                      coders.ProtoCoder,
-                     coders.TimestampCoder,
                      coders.ToStringCoder,
-                     coders.WindowCoder,
-                     coders.WindowedValueCoder])
+                     coders.WindowCoder])
     assert not standard - cls.seen, standard - cls.seen
```

Comment:
We want to make sure at least those coders are tested, but it shouldn't be an error to use other coders along the way.